### PR TITLE
Removed env vars from perf-test dockerfile.

### PR DESCRIPTION
**Pull request by jimleroyer**
# Summary | Résumé

* Removed settings of environment variables within the perf-test Dockerfile setup. These should be unnecessary now as we set these env vars through Terraform passing it to the ECS task definition.
* Removed default Notify resources' IDs from Locust code. These can be configured separately on a locust configuration file for the user and we will minimize exposing test IDs not normally accessible to the public (i.e. it's not bad but not ideal and we can configure this in some other way).

# Test instructions | Instructions pour tester la modification

* Run the docker image by passing by the env vars through the command line parameters, as this is what the ECS task will do.

# Release Instructions | Instructions pour le déploiement

This should not break anything as the perf test is already broken as we revoked past referred API key in the header.

# Reviewer checklist | Liste de vérification du réviseur

This is a suggested checklist of questions reviewers might ask during their
review | Voici une suggestion de liste de vérification comprenant des questions
que les réviseurs pourraient poser pendant leur examen :

- [ ] Is the code maintainable? | Est-ce que le code peut être maintenu?
- [ ] Have you tested it? | L’avez-vous testé?
- [ ] Are there automated tests? | Y a-t-il des tests automatisés?
- [ ] Does this cause automated test coverage to drop? | Est-ce que ça entraîne
      une baisse de la quantité de code couvert par les tests automatisés?
- [ ] Does this break existing functionality? | Est-ce que ça brise une
      fonctionnalité existante?
- [ ] Does this change the privacy policy? | Est-ce que ça entraîne une
      modification de la politique de confidentialité?
- [ ] Does this introduce any security concerns? | Est-ce que ça introduit des
      préoccupations liées à la sécurité?
- [ ] Does this significantly alter performance? | Est-ce que ça modifie de
      façon importante la performance?
- [ ] What is the risk level of using added dependencies? | Quel est le degré de
      risque d’utiliser des dépendances ajoutées?
- [ ] Should any documentation be updated as a result of this? (i.e. README
      setup, etc.) | Faudra-t-il mettre à jour la documentation à la suite de ce
      changement (fichier README, etc.)?

### CODE REVIEW

File: tests-perf/ops/Dockerfile [hunk 1]
```diff
@@ -19,17 +19,6 @@ RUN echo "fs.file-max = 100000" >> /etc/sysctl.conf
...
-    PERF_TEST_EMAIL=success@simulator.amazonses.com \
-    PERF_TEST_AWS_S3_BUCKET=notify-perfortmance-test-results-staging \ 
-    PERF_TEST_CSV_DIRECTORY_PATH=/tmp/notify_performance_test \
-    PERF_TEST_DOMAIN=https://api.staging.notification.cdssandbox.xyz \
-    PERF_TEST_SMS_TEMPLATE_ID=83d01f06-a818-4134-bd69-ce90a2949280 \
-    PERF_TEST_BULK_EMAIL_TEMPLATE_ID=5ebee3b7-63c0-4052-a8cb-387b818df627 \
-    PERF_TEST_EMAIL_TEMPLATE_ID=a59b313d-8de2-4973-ac2f-66de7ec0b239 \
-    PERF_TEST_EMAIL_WITH_ATTACHMENT_TEMPLATE_ID=a59b313d-8de2-4973-ac2f-66de7ec0b239 \
-    PERF_TEST_EMAIL_WITH_LINK_TEMPLATE_ID=5ebee3b7-63c0-4052-a8cb-387b818df627 \
-    TEST_AUTH_HEADER="ApiKey-v1 c55039fc-c0e1-44db-a14e-b6a669148ec6"
```
Comment (sastels) [inline tests-perf/ops/Dockerfile:33]: Hi guys to remember how docker works… do we need an ARG for each of theses and then something like the GIT_SHA=${GIT_SHA}

Comment (jimleroyer) [reply tests-perf/ops/Dockerfile:33]: Environment variable would be defined [via the `--env` parameter on the command line](https://docs.docker.com/engine/reference/commandline/run/#set-environment-variables--e---env---env-file).

Regarding the GIT_SHA, I do wonder if we need to keep that one in there. It looks like setting up an env var using an env var. I can remove that too.

Comment (sastels) [reply tests-perf/ops/Dockerfile:33]: I think it’s moving the vars from what’s passed to docker to what’s in the image? Though… in our case we’re gonna inject them with AWS, not when the image is built. So yeah I think what’s here is right.

**Review by sastels — APPROVED** [review]
🚀

File: tests-perf/ops/Dockerfile (hunk 1 above)
Comment (jimleroyer) [reply tests-perf/ops/Dockerfile:33]: Yes you are right about the `arg`, I didn't notice. But removing both arg and env setup. 👍

**Pull request merged**